indy-besu/vdr/src/client/implementation/web3/client.rs

```diff
@@ -26,10 +27,14 @@ impl Web3Client {
     pub fn new(node_address: &str, signer: Option<Box<dyn Signer>>) -> VdrResult<Web3Client> {
         let transport = Http::new(node_address).map_err(|_| VdrError::ClientNodeUnreachable)?;
         let web3 = Web3::new(transport);
-        Ok(Web3Client {
+        let web3_client = Web3Client {
             client: web3,
             signer,
-        })
+        };
+
+        debug!("Created new Web3Client");
```

Comment:
Can we log additional information about the created Web3Client?

Reply:
Added node_address